packages/solid-keyring/src/providers/Keyring.ts

```diff
@@ -31,7 +36,7 @@ const defaultState: KeyringContextState = {
   space: undefined,
   spaces: [],
   agent: undefined,
-  account: undefined
+  account: window.localStorage.getItem(W3UI_ACCOUNT_LOCALSTORAGE_KEY) ?? undefined
```

Comment:
Can we make this a property on `<KeyringProvider>` so we can abstract away any localStorage details?
user of keyring can store this state however they want, as long as they pass it as something like `<KeyringProvider account={getItem(W3UI_ACCOUNT_LOCALSTORAGE_KEY)}>...`

Reply:
if we did that we'd need to require users of this library to also hand us a setter, so it would be something like:

```
<KeyringProvider account={getItem(W3UI_ACCOUNT_LOCALSTORAGE_KEY)} setAccount={(account) => {window.local.setItem(W3UI_ACCOUNT_LOCALSTORAGE_KEY, account)}}>
```

I don't think this is a very friendly to the users of this library - maintaining auth state across page refreshes feels like a core part of what we're trying to offer with these components and requiring a user to think about where they want to store the currently logged in account feels like a big ask - especially for new developers who probably don't particularly know where they want to store this state.

I'd definitely be open to making this an option, but I'd prefer to wait until we have at least one use-case (internal or external) that calls for it.

Reply:
We talked about this in person and both seem fine to do something that relies on `window.localStorage`, and if that leads to an error, we'll design something better in the context of that use case. :)

Reply:
I realized today that anything using these libraries will likely need to disable SSR, because the primary functionality we're delivering is managing client-side state. 

This manifests in errors like these that point out that you can't use `createContext` in components that render server-side - even if we did decide not to use `window.localStorage` in this lib it will never work in Node:

<img width="986" alt="Screenshot 2023-04-07 at 2 53 18 PM" src="https://user-images.githubusercontent.com/1113/230683803-ced17710-7742-4b98-90ba-083ce3284698.png">